spec/models/topic_spec.rb

```diff
@@ -0,0 +1,8 @@
+require 'rails_helper'
+
+describe 'Topic', type: :model do
+  describe 'Validations' do
+    subject { build :user }
```

Comment:
Shouldn't it be topic instead of user? And you should validate the presence of name